package.json

```diff
@@ -477,6 +478,7 @@
 		"cheerio": "^1.0.0-rc.3",
 		"express": "^4.17.1",
 		"hasbin": "^1.2.3",
+		"lean-client-js-core": "^1.2.12",
```

Comment:
NPM doesn't seem to make this available just because it's a dependency of "lean-core-js-node", so I added it explicitly here.